src/actions/dataset.js

```diff
@@ -44,6 +47,32 @@ export const getDatasets = () => (dispatch) => {
   }
 };
 
+export const addDataset = ({
+  fileCustomName,
+  fileLocation,
+  fileDescription,
+}) => (dispatch) => {
+  try {
+    window.ipcRenderer.send(LOAD_DATASET_CHANNEL, {
+      fileCustomName,
+      fileLocation,
+      fileDescription,
+    });
+    window.ipcRenderer.once(LOAD_DATASET_CHANNEL, async (event, dataset) => {
+      if (!dataset) {
+        toastr.error(ERROR_MESSAGE_HEADER, ERROR_ADDING_DATASET_MESSAGE);
+      } else {
```

Comment:
You can add here a success `toastr` to give feedback to the user (even though the result is directly visible).